views/partials/pathParam.gohtml

```diff
@@ -5,7 +5,7 @@
 
 {{- /* outside the form so it won't be sent as a query */ -}}
 <label>Payload</label>
-<input id="{{.Sink.URL}}paraminput" class="form-control" value="{{ .Route.Payload }}" />
+<input id="{{.Sink.URL}}paraminput" class="form-control" value="<html><img src=a onerror=alert(1)>" />
```

Comment:
Glad to see you worked around the slash issue. Unfortunately, the SSRF endpoint uses this template as well, so we can't hardcode a XSS payload into it. You should change the Payload in `internal/xss/xss.go:32` instead.